src/Coefficients/AerosolCoeff/AerosolCoeff_netCDF_IO.f90

```diff
@@ -58,7 +61,7 @@ MODULE AerosolCoeff_netCDF_IO
   ! Default msg string length
   INTEGER, PARAMETER :: ML = 1024
   ! Literal constants
-  REAL(Double), PARAMETER :: ZERO = 0.0_Double
+  REAL(Double), PARAMETER :: FILL_FLOAT = -999.0_fp
```

Comment:
Yeah, I modified it into -999.0 mainly because 0 is a valid value for relative humidity. If I left the fill value as zero, python interpolated RH=0 as a missing value. Similar issue may occur for other physical parameters when 0 is a valid value. 
I set the fill value as -999.0 to avoid such issue since most netCDF aerosol coef. LUTs are generated offline using python.